frontend/catalyst/pennylane_extensions.py

```diff
@@ -2555,6 +2560,42 @@ def _get_batch_size(args_flat, axes_flat, axis_size):
     return batch_size
 
 
+class CallbackClosure:
+    """This is just a class containing data that is important for the callback."""
+
+    def __init__(self, *absargs, **abskwargs):
+        self.absargs = absargs
+        self.abskwargs = abskwargs
+
+    @property
+    def tree_flatten(self):
+        """Flatten args and kwargs."""
+        return tree_flatten((self.absargs, self.abskwargs))
+
+    @property
+    def getLowLevelSignature(self):
```

Comment:
Minor: As it is a property, I would remove the get

Reply:
https://github.com/PennyLaneAI/catalyst/pull/596/commits/0948fbd55e244c92171bd250ad37cf5dd32b71e3